packages/form-validations/partials/header.hbs

```diff
@@ -0,0 +1,16 @@
+<header class='bg-inverted text-secondary-inverted py-1000 py-tablet-1100'>
+  <div class='Container'>
+
+    <h1 class='typography-heading-xlarge-text'>Spirit Design System</h1>
+    <h2 class='typography-heading-medium-text text-secondary-inverted-disabled mb-900'>Form Validations — Development
+      Preview</h2>
+
+    <a href='https://github.com/lmc-eu/spirit-design-system' class='Button Button--inverted Button--large mr-400'>
+      GitHub Project
+    </a>
+    <a href='https://spirit.supernova-docs.io/spirit/' class='Button Button--inverted Button--large'>
+      Spirit Docs →
+    </a>
+
+  </div>
+</header>
```

Comment:
No end empty line. 👀

Reply:
I see this file as `LF` in my IDE. As it is configurated in `.editorconfig`.